pillar.example

```diff
@@ -1,6 +1,10 @@
 vault:
-  lookup:
-    pkg: myvault
-    conf_file: /usr/local/etc/vault/vault.conf
-  config:
-    key: othervalue
+  overrides:
+    keybase_users:
+      - renaissancedev
+      - pdpinch
+      - bdero
+    secret_shares: 3
+    secret_threshold: 2
+
+vault.verify: False
```

Comment:
The vault.verify setting is picked up by the execution module to specify whether to validate the TLS certificate used by Vault.